hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMClientProtocolServer.java

```diff
@@ -509,6 +543,104 @@ public List<ContainerInfo> listContainer(long startContainerID,
     }
   }
 
+  /**
+   * Lists a range of containers and get their info.
+   *
+   * @param startContainerID start containerID.
+   * @param count count must be {@literal >} 0.
+   * @param state Container with this state will be returned.
+   * @param factor Container factor.
+   * @return a list of containers capped by max count allowed
+   * in "hdds.container.list.max.count" and total number of containers.
+   * @throws IOException
+   */
+  @Override
+  @Deprecated
```

Comment:
Instead of adding a new method that's already deprecated, we can call the one that accepts `ReplicationConfig`:

https://github.com/apache/ozone/blob/6b2f0f2e8e0541d7175093f0acd73fe7fdff7bab/hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMClientProtocolServer.java#L726-L731